sdk/src/impl/position-impl.ts

```diff
@@ -40,87 +45,118 @@ export class PositionImpl implements Position {
   async increaseLiquidity(
     liquidityInput: IncreaseLiquidityInput,
     sourceWallet?: Address,
-    positionWallet?: Address
+    positionWallet?: Address,
+    payer?: PublicKey
   ) {
     const sourceWalletKey = sourceWallet
       ? AddressUtil.toPubKey(sourceWallet)
       : this.ctx.wallet.publicKey;
     const positionWalletKey = positionWallet
       ? AddressUtil.toPubKey(positionWallet)
       : this.ctx.wallet.publicKey;
+    const payerKey = payer ? payer : this.ctx.wallet.publicKey;
```

Comment:
nit: can use the null coaelsecing operating, `payer ?? this.ctx.wallet.publicKey`